src/test/java/dev/openfeature/javasdk/FlagEvaluationSpecTest.java

```diff
@@ -251,9 +262,10 @@ private Client _client() {
     }
 
     @Specification(number="1.3.3", text="The client SHOULD guarantee the returned value of any typed flag evaluation method is of the expected type. If the value returned by the underlying provider implementation does not match the expected type, it's to be considered abnormal execution, and the supplied default value should be returned.")
+    @Test void type_system_prevents_this() {}
+
     @Specification(number="1.1.6", text="The client creation function MUST NOT throw, or otherwise abnormally terminate.")
-    @Disabled("Not sure how to test?")
-    @Test void not_sure_how_to_test() {}
+    @Test void constructor_does_not_throw() {}
```

Comment:
Just added some explanation here as in other tests as to why these tests aren't really necessary for the Java SDK.